src/tests/work-item/work-item-list/page-objects/common.page.js

```diff
@@ -153,6 +153,12 @@ class CommonPage {
     this.microservicesMenuTab.click(); 
     return new MicroservicesPage();
   }
+  clickMobileViewHeaderMenu () {
+    return element(by.id('header-toggle-menu-btn')).click();
```

Comment:
@ldimaggi , its the toggle icon which is named as btn or header-toggle-menu-btn :)